src/app/split-view/multi/terra-multi-split-view.component.ts

```diff
@@ -307,34 +305,22 @@ export class TerraMultiSplitViewComponent implements OnDestroy, OnInit
 
                 // focus view horizontally
                 if(anchor[0] != null &&
-                   anchor[0].getBoundingClientRect().left > viewContainer.scrollLeft() - offset &&
-                   anchor[0].getBoundingClientRect().right <= viewContainer[0].getBoundingClientRect().right)
+                   anchor[0].getBoundingClientRect().left > viewContainer.offset().left &&
+                   anchor[0].getBoundingClientRect().right <= viewContainer.offset().left + viewContainer.outerWidth())
                 {
                     return;
                 }
 
-                // offset fix for navigator
-                if(prevSplitView[0] != null)
-                {
-                    offset = offset + prevSplitView.width() + (3 * offset);
-                }
-
-                // offset fix for overlay
-                if($($(anchor[0].closest('.hasSplitView')).find(anchor))[0] != null)
```

Comment:
Hast du das mal mit einer Split-View im Overlay getestet? Wenn ja, wo gibt es eine, damit ich mir das auch mal anschauen kann!?

Reply:
Nein, mir ist hier nur der Anwendungsfall in der Log-View bekannt & dort wird noch die alte Splitview verwendet. Kläre die Prio hier nochmal mit Jens ab.